File: source/script/ScriptLoader.java
package script;

public class ScriptLoader extends ClassLoader {
    public ScriptLoader() {}

    public Class<?> load(String name, byte[] data) {
        defineClass(name, data, 0, data.length);

        try {
            return loadClass(name);
        }
        catch (Exception ignored) {
            return null;
        }
    }
}

File: source/script/ScriptLoader.kt
package script

class ScriptLoader : ClassLoader() {
    fun load(name: String, data: ByteArray): Class<*>? {
        defineClass(name, data, 0, data.size)

        return try {
            loadClass(name)
        }
        catch (ignored: Exception) {
            null
        }
    }
}